src/features/authService/auth.service.js

```diff
@@ -15,11 +15,15 @@ export default function ($rootScope, $http, $location, $sessionStorage, $base64,
         }).then(function (res) {
             user.id = res.data;
             const authdata = $base64.encode(user.username + ':' + user.password);
-            $sessionStorage.putObject('user', {username: user.username, id: user.id, auth: authdata});
+            if(fbAuth == true){
+                $sessionStorage.putObject('user', {username: user.username, id: user.id, auth: authdata,pass : password});
```

Comment:
a dałoby radę zrobić to tak żeby nie trzymać hasła w wersji jawnej? bo tak trochu średnio to wygląda jak sie wejdzie na narzędzia programisty ;)